src/NuGetPackageConfigConverter/Services/PackageManagerConverter.cs

```diff
@@ -272,4 +392,53 @@ private static ProjectItem GetProjectItem(ProjectItems items, string name)
             return null;
         }
     }
+
+
+
+    static class ProjectExtensions
+    {
+        public static string GetFullName(this Project project)
+        {
+            try
+            {
+                return project.FullName;
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+        }
+    }
+
+
+    public interface ILogger
+    {
+        void Log(string txt);
+    }
+
+    public class Logger : ILogger
+    {
+        private readonly StreamWriter logfile;
+        public Logger(Solution sln)
```

Comment:
I'd prefer for this to be written to the output window. I can take care of that after it's merged in, though

Reply:
It can be written to the output window too, but *when* the vsix crashes then the output window is gone, and then it is very nice to have the file log.